### PR TITLE
build: Add six requirement to install requirements

### LINKED ISSUE #35: Specify `six` dependency
Coming here from https://github.com/recast-hep/recast-atlas/issues/76, `yadageschemas` uses `six` in

https://github.com/yadage/yadage-schemas/blob/260fbc40ec5b8afa7d59c0dd00591e02e13774e4/yadageschemas/dialects/raw_with_defaults.py#L118

but doesn't specify this in its requirements

https://github.com/yadage/yadage-schemas/blob/260fbc40ec5b8afa7d59c0dd00591e02e13774e4/setup.py#L11-L17

This creates problems upstream.

### PR DESCRIPTION
**Pull request by matthewfeickert**
Resolves #35

The `six.moves` module 

https://github.com/yadage/yadage-schemas/blob/260fbc40ec5b8afa7d59c0dd00591e02e13774e4/yadageschemas/dialects/raw_with_defaults.py#L118

was added in `six` `v1.4.0`, setting the lower bound of `six>=1.4.0`.

### CODE REVIEW

**Comment by matthewfeickert** [comment]
@lukasheinrich gentle ping on this.

**Comment by matthewfeickert** [comment]
@lukasheinrich If you can review this, and once merged release a new patch version `v0.10.7` to PyPI, that would help things downstream for `recast-atlas` (c.f. https://github.com/recast-hep/recast-atlas/issues/76).

**Review by lukasheinrich — APPROVED** [review]

**Comment by matthewfeickert** [comment]
@lukasheinrich As I'm not a member of the yadage GitHub organization, and so don't have commit rights, can you merge this PR now that it has been approved?

After this is merged if you can also create a `v0.10.6` patch release of `yadage-schemas` and put it up on PyPI that would address the last bits of Issue #35.

**Pull request merged**
